temel_bilgiler/javascript_temelleri/fundamentals-1.md

```diff
@@ -0,0 +1,118 @@
+### Giriş
+JavaScript'e dalalım!
+
+### Ders Özeti
+Bu bölümde öğreneceğiniz konuların genel bir özeti bulunmaktadır.
+
+* Bir değişken nasıl tanımlanır?
+* Bir değişkeni tanımlamanın üç farklı yolu nelerdir?
+* Hangi birini ne zaman kullanmalısınız?
+* Değişkenler için adlandırma kuralları nelerdir?
+* Operatörler, işlem yapılandırıcıları ve işlemler nelerdir?
+* Birleştirme nedir ve sayıları ve dizeleri birleştirirken ne olur?
+* JavaScript'teki farklı operatör türleri nelerdir?
+* == ve === arasındaki fark nedir?
+* Operatör öncelik değerleri nelerdir?
+* Artırma / azaltma operatörleri nelerdir?
+* Bunları öne eklemenin ve sonuna eklemenin farkı nedir?
+* Atama operatörleri nelerdir?
+* Unary Plus Operator nedir?
+
+### JavaScript Kodunu Nasıl Çalıştırılır?
+
+Fundamentals kursunun çoğunda yazacağımız tüm JavaScript, tarayıcı üzerinden çalıştırılacaktır. Fundamentals ve NodeJS yolu sonraki derslerinde JavaScript'i tarayıcı ortamının dışında nasıl çalıştıracağınızı gösterecektir. Bu derslerin dışında, şimdilik istisna belirtilmedikçe JavaScript'inizi tarayıcıda çalıştırmaya her zaman varsaymalısınız, aksi takdirde beklenmedik hatalarla karşılaşabilirsiniz.
```

Comment:
fix: Temel bilgiler kursunda yazacağımız JavaScript kodunun çoğunlukla tarayıcı üzerinde çalıştıracağız. JavaScript'in tarayıcı dışındaki bir ortamda nasıl çalıştırılacağını, ilerleyen zamanda temel bilgiler ve NodeJS kurslarında öğreneceksiniz. O zamana kadar tam tersi belirtilmedikçe JavaScript'i tarayıcınızda çalıştırmalısınız, aksi takdirde beklenmedik hatalarla karşılaşabilirsiniz.